bootstrap/kubeadm/internal/cloudinit/controlplane_join.go

```diff
@@ -35,6 +35,9 @@ runcmd:
 {{- template "commands" .PostKubeadmCommands }}
 {{- template "ntp" .NTP }}
 {{- template "users" .Users }}
+{{- template "disk_setup" .DiskSetup}}
+{{- template "fs_setup" .DiskSetup}}
+{{- template "mounts" .Mounts}}
```

Comment:
Should these also be added to the `nodeCloudInit` template in `bootstrap/kubeadm/internal/cloudinit/node.go` as well?

Reply:
Yes good catch! I only added them to control planes in the prototype because I was testing with etcd but there can definitely be a use case for disk mounts with node machines as well.

Reply:
/hold

Reply:
fixed 

/hold cancel